tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/InstallationDetailsTest.java

```diff
@@ -0,0 +1,186 @@
+			"jboss\\.discovery\\.earlyaccess\\.site\\.integration-stack\\.url\\|devstudio\\|{majorVersion}.*=https://devstudio\\.redhat\\.com/{majorVersion}.*/development/updates/integration-stack/discovery/earlyaccess/",
+			"jboss\\.discovery\\.site\\.integration-stack-sap\\.url\\|devstudio\\|{majorVersion}.*=https://devstudio\\.redhat\\.com/{majorVersion}.*/development/updates/integration-stack/extras/" };
+
+	public String getJbossToolsMajorVersion() {
+		return jbossToolsVersion.substring(0, jbossToolsVersion.indexOf('.'));
+	}
+
+	@Before
+	public void setup() {
+		log.info("Navigate to installation details");
+		new WorkbenchShell().maximize();
+		new ShellMenuItem(HELP_BUTTON, ABOUT_MENU_BUTTON).select();
+
+		// get styled text from help
+		DefaultStyledText dt = new DefaultStyledText();
+		String text = dt.getText();
+
+		// TODO use
+		// org.eclipse.core.runtime.Platform.getBundle("org.jbosstools").getVersion();
+		int versionIndex = 0;
+		jbossToolsVersion = text.substring(versionIndex = text.indexOf("Version:") + 9,
+				text.indexOf("Build id", ++versionIndex));
+		log.info("JBossTools version: " + jbossToolsVersion);
+
+		assertFalse("The JBossTools version value is empty!", jbossToolsVersion.isEmpty());
+
+		new PushButton(MODAL_DIALOG_ABOUT_DEVSTUDIO_INSTALLATION_DETAILS_BUTTON).click();
+
+		TabFolder tabFolder = new DefaultTabItem(MODAL_DIALOG_CONFIG_MENU_BUTTON).getTabFolder();
+
+		Optional<TabItem> configurationItem = tabFolder.getItems().stream()
+				.filter(i -> i.getText().matches(MODAL_DIALOG_CONFIG_MENU_BUTTON)).findAny();
+
+		assertTrue("Tab item '" + MODAL_DIALOG_CONFIG_MENU_BUTTON + "' is missing", configurationItem.isPresent());
+		TabItem tabItem = configurationItem.get();
+		tabItem.activate();
+
+		new WaitWhile(new JobIsRunning(new RegexMatcher("Configuration Fetch Job"), true), TimePeriod.LONG);
+	}
+
+	// JBDS-5064
+	@Test
+	public void validateLinksWithGlobalConfig() {
+		log.info("Navigate to '" + MODAL_DIALOG_CONFIG_MENU_BUTTON + "' and validate links with global config");
+		DefaultText dt = new DefaultText();
+		String localText = dt.getText();
+		log.debug("Text of global config:\n" + localText);
+		ConfigPropertiesHelper rcph = new ConfigPropertiesHelper(localText);
+
+		// note that the version substring may change with devstudio version
+		String majorVersion = getJbossToolsMajorVersion();
+		log.info("Major version: " + majorVersion);
+
+		Arrays.stream(regexArray).forEach(regexRaw -> {
+			String regex = regexRaw.replace("{majorVersion}", majorVersion);
+			log.info("Validating local and remote config with regex: '" + regex + "'");
+			boolean result = rcph.isRegexValid(regex);
+			assertTrue("Unable to match regex '" + regex + "' to either local or remote config with result: " + result,
+					result);
+		});
+
+		log.info("Finished: validateLinksWithGlobalConfig");
+
```

Comment:
you should close all opened shells after the test. Otherwise it might break following tests. You can simply use WorkbenchShellHandler.getInstance().closeAllNonWorbenchShells()